package.json

```diff
@@ -34,11 +35,13 @@
   },
   "homepage": "https://github.com/javascriptair/javascriptair.github.io#readme",
   "devDependencies": {
+    "autoprefixer": "^6.2.1",
```

Comment:
One thing I'll change after this is merged is I'll remove these semver range symbols. [Because this](https://medium.com/@kentcdodds/why-semver-ranges-are-literally-the-worst-817cdcb09277) :-)